src/Components/signUpPage.scss

```diff
@@ -0,0 +1,189 @@
+@import "../styles/main.scss";
+.signUp {
+  height: 700px;
+  width: 100%;
+  background-color: $black;
+  @include flex(column, center, center);
+  &__form {
+    @include flex(column, center, center);
+    width: 400px;
+    height: 500px;
+  }
+  &__wrapper {
+    @include flex(column, space-around, center);
+    width: 450px;
+    height: 100%;
+    background-color: $background-color;
+    border: 1px solid $border-color;
+    border-radius: 5px;
+    padding: 3rem;
```

Comment:
margin and paddings should be pixels not rem, change every where you used rems for margin and paddings to pixels